apps/api/e2e/tests/voting.test.ts

```diff
@@ -66,6 +66,11 @@ describe("Voting API", () => {
       const competitionResponse = await createTestCompetition(
         adminClient,
         competitionName,
+        undefined, // description
+        undefined, // externalUrl
+        undefined, // imageUrl
+        undefined, // competition type
+        new Date().toISOString(), // start date to allow voting
```

Comment:
This test is asserting that voting works, so we need to enable it for the competition.  You'll see this type of change in a few places, as well as a change to the test helper to support setting a voting start date.